pkg/cli/store.go

```diff
@@ -85,6 +85,7 @@ func init() {
 	set.String("rest-api-server-port", "8081", "port to serve the REST API from")
 	set.String("rest-api-tls-cert-file", "", "path to the TLS certificate in PEM format for rest api server")
 	set.String("rest-api-tls-key-file", "", "path to the TLS key in PEM format for rest api server")
+	set.Bool("db-direct-connection", false, "connect directly to the database that backs the gql API for optimized endpoint implementations")
```

Comment:
hmm (ignorant) question on maintaining this separate direct db connection - how will this change with graphQL changes and ent usage? will we have to maintain 2 separate implementations?

Reply:
+1 on @mihaimaruseac 's suggestion on read-only.. One thing i'd want to add is perhaps an experimental comment on this feature, unless we all vote on this being a supported path going forward. I can see the cost of maintaining this feature being significant in sync and not drifting. I'd put a warning on the use of the feature.

If we do decide that we want to support this then we have to add it to the PR template that all changes need to be checked here as well. (which i don't think we are deciding yet).... 

tl;dr; i'd lgtm if we add experimental and have a warning banner when users use this. Otherwise we should discuss this at the maintainers meeting.

Reply:
I agree, added a log and modified the flag description to reflect this!